src/components/Viewer/Properties/Rendering.tsx

```diff
@@ -0,0 +1,22 @@
+import { PrimitivesExternalWebResource } from "src/types/primitives";
+import { Rendering } from "src/components/Primitives";
+import React from "react";
+
+interface PropertiesRenderingProps {
+  rendering: PrimitivesExternalWebResource[];
+}
+
+const PropertiesRendering: React.FC<PropertiesRenderingProps> = ({
+  rendering,
+}) => {
+  if (rendering?.length === 0) return <></>;
+
+  return (
+    <>
+      <span className="manifest-property-title">Rendering</span>
```

Comment:
I opened my manifest in some other viewers and started to second guess the label of the span here. Mirador calls this, "Alternate formats." Do you think that is better?  Rendering means nothing to me as a user.

Reply:
Yeah, let's copy Mirador here. This should intersect nicely with Adam's work in #190 too once that gets pulled through.